docs/src/main/sphinx/release/release-460.md

```diff
@@ -0,0 +1,52 @@
+# Release 460 (2 Oct 2024)
+
+## Atop connector
+
+* {{breaking}} Remove the Atop connector. ({issue}`23550`)
+
+## ClickHouse connector
+
+* Improve performance of listing columns. ({issue}`23429`)
+* Improve performance for queries comparing `varchar` columns. ({issue}`23558`) 
+* Improve performance for queries using `varchar` columns for `IN` comparisons. ({issue}`23581`)
+* Improve performance for queries with complex expression for `LIKE` clauses. ({issue}`23591`)
+
+## Delta Lake connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
```

Comment:
What's fixed? How was it broken?

Reply:
I think it relates to some incompatibility with dictionaries that are larger than one page. Not sure how to phrase that so users understand or its vague enough

Reply:
What was the failure mode? Missing bloom filters? Broken parquet files? Query failures?

Reply:
Alter table execute optimize failed from the reported issue, but I think there might have been others as well. Maybe

Prevent query failures when writing Parquet files with Bloom filter on columns.

But I am honestly not sure that captures it correctly as well.

Reply:
Any insert query could fail when the writer for a column configured with a bloom filter would fall back from dictionary encoding to plain encoding due to large cardinality.
We didn't produce broken files as the failure prevents the insert from completing.

Reply:
Or simply, "Fix query failures when writing bloom filters in Parquet files"

Reply:
Done